packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -1,12 +1,60 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    linkActiveFontColor,
+    linkFontColor,
+    smallPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        inline-size: auto;
+        overflow: auto;
+        block-size: 100%;
+        min-block-size: 36px;
+    }
+
+    :host([fit-to-content]) {
+        block-size: auto;
+    }
+
+    .container {
+        font: inherit;
+        outline: none;
+        box-sizing: border-box;
+        position: relative;
+        color: inherit;
+        min-inline-size: 100px;
+        padding: ${smallPadding};
+        margin-block-end: 10px;
+        margin-inline-end: 10px;
+    }
+
+    .container > :first-child {
+        margin-block-start: 0;
+    }
+
+    .container > :last-child {
+        margin-block-end: 0;
+    }
+
+    li > p {
+        margin-block: 0;
+    }
+
+    a {
+        word-break: break-all;
+        color: ${linkFontColor};
+    }
+
+    a:active {
```

Comment:
Could you add hover styling to match the Nimble anchor too?

Reply:
The default nimble anchor link doesn't have any hover styling. Only if the `underline-hidden` attribute is enabled, on hovering the link, underline appears. Also, there is no specific font color for hovering over the [nimble anchor](https://nimble.ni.dev/storybook/?path=/docs/components-anchor--docs).

Reply:
My bad, I clearly made this comment in a rush. Thanks for checking.